gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon Dec 28 10:00:20 PST 2015
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```

Comment:
those files are not excluded by `.gitignore` so should be added to repo
+ it indicates current gradle version